### ISSUE TITLE
Can the NISAR track-frames be shared publicly on sample product page?

### ISSUE THREAD

**Issue by alex-s-gardner** [isce-framework/nisar-workflows#17]
It would be useful if people outside of the project could get access to the NISAR track-frame convention.

**Comment by hfattahi** [comment]
Thanks @alex-s-gardner for bringing this up. A version of the NISAR track-frame database have been available [on this website](https://nasa.maps.arcgis.com/apps/webappviewer/index.html?id=3ec81e1c6439470e9dee31dbd600afe1) in the past few years. We have been discussing adding information about the observation plan to this arc map. But yes I assume the actual database file can be also useful.

**Comment by alex-s-gardner** [comment]
Thank @hfattahi... I hadn't seen that site before. Looks very useful.  Is the data downloadable from that site (I couldn't seem to get the data) and is that track-frame database up-to-date?

I have a the latests copy from the g-drive... just wondering about public access.

**Comment by hfattahi** [comment]
> Thank @hfattahi... I hadn't seen that site before. Looks very useful. Is the data downloadable from that site (I couldn't seem to get the data) and is that track-frame database up-to-date?
> 
> I have a the latests copy from the g-drive... just wondering about public access.

Yeah I doubt if the database is downloadable here. I agree with you that we should make it available to the users. Yes I think the track and frame numbers are consistent with what you have. But anyways there is a plan to update this with additional information about the observation plan.
